.github/scripts/intake_to_linear.py

```
@@ -0,0 +1,161 @@
+        key = name.strip().lower()
+        lid = label_index.get(key) or label_index.get(key.split("/")[-1])
+        (label_ids.append(lid) if lid else unmatched_labels.append(name))
+
+    # --- map project by name (fuzzy contains) ---
+    project_id, project_note = None, ""
+    want = (s.get("project") or "").strip().lower()
+    if want and want != "standalone":
+        projs = gql("{ projects(first:200){ nodes{ id name } } }")["projects"]["nodes"]
+        for p in projs:
+            if want == p["name"].lower() or want in p["name"].lower() or p["name"].lower() in want:
+                project_id = p["id"]; break
```

Ruff (0.15.18)

[error] Multiple statements on one line (semicolon)

(E702)